src/app/components/client/toolbar/AnnouncementDialog.tsx

```diff
@@ -461,6 +461,8 @@ export const LocalizedAnnouncementString = (
   // trigger any additional render complexity.
   const parsedString = l10n.getString(key, {
     bundleMonthlyPrice: `$${billingInfo.bundle.monthly}`,
+    bundleYearlyPrice: `$${billingInfo.bundle.individual}`,
+    regularYearlyPrice: `$${billingInfo.yearly}`,
```

Comment:
Shouldn’t this be:

```suggestion
    bundleYearlyPrice: `$${billingInfo.bundle.monthly * 12}`,
    regularYearlyPrice: `$${billingInfo.bundle.individual * 12}`,
```

Reply:
I suppose `regularYearlyPrice` should be the yearly bundle price. Right now it is assigned to the yearly price of Monitor Plus.

Reply:
No, `SUBSCRIPTION_BILLING_AMOUNT_BUNDLE_INDIVIDUAL_MONTHLY_US` is what a user pays for the bundle subscription per month and `SUBSCRIPTION_BILLING_AMOUNT_YEARLY_US` is what a Monitor Plus user pays per month when they have a yearly subscription.